community/learn/graphql-tutorials/tutorials/react-native-apollo/tutorial-site/content/update-delete-mutations.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Update mutations and automatic cache updates"
-metaTitle: "Optimistic UI updates after mutations | GraphQL React Native Apollo Tutorial"
-metaDescription: "We will use the Apollo Client Optimistic Response to perform UI updates after a GraphQL mutation in the React Native app"
+metaTitle: "Automatic cache updates after mutations | GraphQL React Native Apollo Tutorial"
```

Comment:
Is this relevant to this PR?

Reply:
Not really. But it's a single line bug. We can merge it.

Reply:
Okay.